Cargo.toml

```diff
@@ -54,11 +54,8 @@ core-foundation = "0.7"
 core-graphics = "0.19"
 core-text = "15"
 
-[target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios")))'.dependencies]
-freetype = "^0.4.1"
-
 [target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios", target_arch = "wasm32")))'.dependencies]
-servo-fontconfig = "0.4"
+freetype = "^0.5.1"
```

Comment:
Looks like this got messed up - we still need to depend on fontconfig, and the freetype dependency should still be conditional.